10/alpine/Dockerfile

```diff
@@ -1,17 +1,13 @@
 # vim:set ft=dockerfile:
-FROM alpine:3.10
+FROM alpine:3.11
 
-# alpine includes "postgres" user/group in base install
-#   /etc/passwd:22:postgres:x:70:70::/var/lib/postgresql:/bin/sh
-#   /etc/group:34:postgres:x:70:
-# the home directory for the postgres user, however, is not created by default
-# see https://github.com/docker-library/postgres/issues/274
-RUN set -ex; \
-	postgresHome="$(getent passwd postgres)"; \
-	postgresHome="$(echo "$postgresHome" | cut -d: -f6)"; \
-	[ "$postgresHome" = '/var/lib/postgresql' ]; \
-	mkdir -p "$postgresHome"; \
-	chown -R postgres:postgres "$postgresHome"
+# 70 is the standard uid/gid for "postgres" in Alpine
```

Comment:
Would it make sense to unify explicit user ids across the Debian and Alpine images, like [redis](https://github.com/docker-library/redis/pull/203/files#diff-d3ac17aae384a7e89cc0b7645bd38cacR4) and in #93? Can the database directory be used by Alpine if it was created by Debian and vise-versa?

Reply:
In theory, but I think locale issues will wreak havoc on anyone who does so.